src/Parametron.ts

```diff
@@ -315,6 +301,6 @@ export class Parametron {
       })
     })
 
-    return this.pact
+    return this.pact as unknown as Promise<IParametronData>
```

Comment:
Why such monster?

Didn't just `this.pact = new Promise<IParametronData>(...` work?

Reply:
👹 👹 👹 👹